src/SourceBuild/content/CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+Contributing
+============
+
+See [dotnet/runtime Contributing](./src/runtime/CONTRIBUTING.md) for information about coding styles, source structure, making pull requests, and more.
```

Comment:
```suggestion
See contributing guidelines in the individual repos for information about coding styles, source structure, making pull requests, and more.
[dotnet/runtime](./src/runtime/CONTRIBUTING.md) has general contribution guidelines such as allowed licenses.
```
Individual repos have different coding styles, source structure, etc. We want to link to dotnet/runtime just for general information like licenses.